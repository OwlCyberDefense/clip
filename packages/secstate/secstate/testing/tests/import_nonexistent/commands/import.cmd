/usr/sbin/secstate import /root/notfound.xml
