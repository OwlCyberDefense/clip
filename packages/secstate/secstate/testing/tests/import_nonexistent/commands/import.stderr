No such file: '/root/notfound.xml'
